{
    "name":             "rescue",
    "version":          "0.0.2",
    "author":           "Alan Gutierrez <alan@prettyrobots.com>",
    "description":      "Restart a long running error-first callback function on error.",
    "keywords":
    [
                        "error", "exception", "callback"
    ],
    "contributors":     [],
    "homepage":         "http://bigeasy.github.io/rescue",
    "bugs":             "https://github.com/bigeasy/rescue/issues",
    "license":          "MIT",
    "main":             "rescue",
    "repository":
    {
                        "type": "git",
                        "url": "http://github.com/bigeasy/rescue.git"
    },
    "dependencies":
    {
                        "operation":            "0.0.2",
                        "cadence":              "0.4.0"
    },
    "devDependencies":
    {
                        "proof":                "0.0.52"
    },
    "scripts":
    {
                        "test": "proof platform win32 && proof test */*/*.t.js || t/test"
    }
}
